geometry/create-and-edit-geometries/src/main/resources/create_and_edit_geometries/main.fxml

```diff
@@ -0,0 +1,177 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+  ~ Copyright 2023 Esri.
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License"); you may not
+  ~ use this file except in compliance with the License. You may obtain a copy of
+  ~ the License at
+  ~
+  ~ http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+  ~ WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+  ~ License for the specific language governing permissions and limitations under
+  ~ the License.
+  -->
+
+<?import com.esri.arcgisruntime.mapping.view.MapView?>
+<?import javafx.geometry.Insets?>
+<?import javafx.scene.control.Button?>
+<?import javafx.scene.control.ComboBox?>
+<?import javafx.scene.control.Label?>
+<?import javafx.scene.control.Separator?>
+<?import javafx.scene.control.Tooltip?>
+<?import javafx.scene.image.ImageView?>
+<?import javafx.scene.image.Image?>
+<?import javafx.scene.layout.Background?>
+<?import javafx.scene.layout.BackgroundFill?>
+<?import javafx.scene.layout.GridPane?>
+<?import javafx.scene.layout.StackPane?>
+<?import javafx.scene.layout.VBox?>
+<?import javafx.scene.paint.Color?>
+<StackPane fx:controller="com.esri.samples.create_and_edit_geometries.CreateAndEditGeometriesController"
+           xmlns:fx="http://javafx.com/fxml"
+           stylesheets="@style.css">
+    <MapView fx:id="mapView"/>
+    <VBox StackPane.alignment="TOP_RIGHT" maxWidth="120" maxHeight="110" styleClass="panel-region"
+          alignment="CENTER" spacing="6">
+        <padding>
+            <Insets topRightBottomLeft="10"/>
+        </padding>
+        <background>
+            <Background>
+                <fills>
+                    <BackgroundFill>
+                        <fill>
+                            <Color opacity="0.3"/>
+                        </fill>
+                    </BackgroundFill>
+                </fills>
+            </Background>
+        </background>
+        <GridPane vgap="6" hgap="6" alignment="CENTER">
+            <Label GridPane.rowIndex="0" GridPane.columnIndex="0" text="Create"/>
+        </GridPane>
+        <GridPane vgap="6" hgap="6" alignment="CENTER">
```

Comment:
So we can actually put this whole UI into a single GridPane and use a property GridPane.columnSpan="2" where something needs to span right across. I'll send you a DM with a diagram. The main bit is to update the rowIndexes, and add the columSpan values where something needs to span across.